src/common/components/repository-row/index.js

```diff
@@ -13,6 +13,13 @@ import { parseGitHubRepoUrl } from '../../helpers/github-url';
 
 import styles from './repositoryRow.css';
 
+const tickIcon = (
+  <img
+    src='http://assets.ubuntu.com/v1/6c395e6d-green-tick.svg'
+    width={ 22 } height={ 16 }
```

Comment:
It would be better to move these to CSS (repositoryRow.css).

```css
.icon {
  width: 22px;
  height: 16px;
}
```

And use `className={styles.icon}` on the img.